interface/package.json

```diff
@@ -16,7 +16,10 @@
     "react-dom": "^18.2.0",
     "react-mic": "^12.4.6",
     "react-scripts": "^3.4.1",
-    "react-spinners": "^0.10.6"
+    "react-spinners": "^0.10.6",
+    "react-wavesurfer": "^0.8.6",
```

Comment:
What is this package? Seems to be archived https://github.com/mspae/react-wavesurfer